src/pool/pool_scalable.c

```diff
@@ -7,6 +7,8 @@
  *
  */
 
+#define _GNU_SOURCE 1
```

Comment:
Shouldn't this be defined on Linux only?

Reply:
After some discussion I think it would be best to define this in cmake

Reply:
Done